cms/djangoapps/contentstore/utils.py

```diff
@@ -429,30 +429,34 @@ def get_visibility_partition_info(xblock):
     Returns: dict
 
     """
-    user_partitions = get_user_partition_info(xblock, schemes=["verification", "cohort"])
-    cohort_partitions = []
-    verification_partitions = []
-    has_selected_groups = False
-    selected_verified_partition_id = None
-
-    # Pre-process the partitions to make it easier to display the UI
-    for p in user_partitions:
-        has_selected = any(g["selected"] for g in p["groups"])
-        has_selected_groups = has_selected_groups or has_selected
-
-        if p["scheme"] == "cohort":
-            cohort_partitions.append(p)
-        elif p["scheme"] == "verification":
```

Comment:
Note to reviewers: in this PR I am purging a lot of the Studio code of any mention of the "verification" scheme, as it should no longer be used. FYI @efischer19